<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Classifier v0.1</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grp_step1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Step 1 - Load your database</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;
</string>
     </property>
     <property name="text">
      <string>Load your dataset:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inp_dataset">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #BDC3C7;
border-radius: 6px;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_choose_dataset">
     <property name="geometry">
      <rect>
       <x>333</x>
       <y>50</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#3498DB;
color:#FFF;
border-radius:6px;</string>
     </property>
     <property name="text">
      <string>Choose...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;</string>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_step1_status">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>89</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;</string>
     </property>
     <property name="text">
      <string>No dataset loaded yet.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_step2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Step 2 - Configuration</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose your class attribute:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rd_first">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>First column</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rd_last">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Last column</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose amount of training data (percent):</string>
     </property>
    </widget>
    <widget class="QSlider" name="sldr_data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_data">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #BDC3C7;
background-color:#FFF;
border-radius: 6px;</string>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="numDigits">
      <number>3</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>50.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>179</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_step2_status">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>179</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;</string>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_split_data">
     <property name="geometry">
      <rect>
       <x>333</x>
       <y>172</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#3498DB;
color:#FFF;
border-radius:6px;</string>
     </property>
     <property name="text">
      <string>Split data</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>107</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="rd_from_first">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>From first of data set</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>How do you want to choose training data:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rd_from_end">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>From end of data set</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rd_from_random">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose random</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_step3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Step 3 - Start Training</string>
    </property>
    <widget class="QLabel" name="lbl_training_amount">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_all_amount">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_begin_train">
     <property name="geometry">
      <rect>
       <x>333</x>
       <y>100</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#3498DB;
color:#FFF;
border-radius:6px;</string>
     </property>
     <property name="text">
      <string>Start Training</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>If you are agree, Click on the Start Training,otherwise change your configuration:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_step3_status">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;</string>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_step4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Step 4 - Start Testing</string>
    </property>
    <widget class="QLabel" name="lbl_step4_info">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Click on Test it! button to see the error rate of test data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_begin_test">
     <property name="geometry">
      <rect>
       <x>333</x>
       <y>70</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#3498DB;
color:#FFF;
border-radius:6px;</string>
     </property>
     <property name="text">
      <string>Test it!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_step4_status">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;</string>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_results">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Results</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predictive accuracy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_predict">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>31</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Confidence Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.95</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Standard error:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_standard">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>We can say with probabilty of 95% the true predictive accuary lies in the interval:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_up_bound">
     <property name="geometry">
      <rect>
       <x>123</x>
       <y>140</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_down_bound">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>140</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_down_bound_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;   x   &lt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About Classifier...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
